react/components/ComponentEditor/index.tsx

```diff
@@ -30,6 +32,50 @@ const defaultUiSchema = {
 
 const MODES: ComponentEditorMode[] = ['content', 'layout']
 
+const nativeMap: Record<string, ComponentSchema> = {
+  nativeImage: {
+    type: 'string',
+    widget: {
+      'ui:widget': 'image-uploader',
+    },
+  },
+  nativeOptions: {
+    type: 'number',
+    widget: {
+      'ui:options': {
+        inline: true,
+      },
+      'ui:widget': 'RadioWidget',
+    },
+  },
+  nativePlainText: {
+    type: 'string',
+    widget:{
+      'ui:widget': 'BaseInput',
+    },
+  },
+  nativeRichText: {
+    type: 'string',
+    widget: {
+      'ui:widget': 'TextareaWidget',
+    },
+  },
+  nativeVideoLink: {
+    type: 'string',
+    widget:{
+      'ui:widget': 'TextareaWidget',
+    },
+  },
+}
+
+const translateFromNative = (schema: ComponentSchema): ComponentSchema => {
```

Comment:
I think this modification is from #119.. Since it wasn't merged to master, I think you should just commit the change related to your feature!